microsoft-service-bus/2.2/modules/ROOT/pages/index.adoc

```diff
@@ -245,6 +245,29 @@ payload]]></ee:set-payload>
 </mule>
 ----
 
+== SSL Issues
+
+You must configure the truststore in insecure mode if you get one of the following errors when performing a connectivity test or starting a Mule app:
+
+* `SSL certificate error`
+* `General SSLEngine problem`
+
+To configure the truststore in insecure mode:
+
+. Open the global element configuration.
```

Comment:
How do you open global element configuration?

Reply:
I see your point, but I don't want to explain how to do that here. I can refer to the Studio section, but then we get into the intra-topic link issue.